docs/user-guide/cli-using-using-profiles.md

```diff
@@ -61,6 +62,14 @@ zowe zos-files list data-set "ibmuser.*" --host myhost123 --port myport123
 
 **Note:** If you choose to log in to Zowe API Mediation Layer, a base profile is created for you to store a web token, host, and port.
 
+### Tips for using base profiles
+
+Use the base profile to share option values between services. You might share option values between services in the following situations:
+- You have multiple services that share the same username, password, or other value.
+- You want to store a web token to access all services through Zowe API Mediation Layer.
+- You want to trust a known self-signed certificate or your site does not have server certificates configured. You can define `reject-unauthorized` in the base profile with a value of false to apply to all services. Understand the security implications of accepting self-signed certificates at your site before you use this method.
+If you have multiple LPARs and want to share option values only between services that run on the same LPAR, you can use nested profiles to achieve this (see Example 1 below).
```

Comment:
Should this last sentence be removed? I think Example 1 is referencing another document.